bears/yml/RAMLLintBear.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='ramllint',
+        output_format='regex',
+        output_regex=r'(?P<severity>error|warning|info).*\n  (?P<message>.+) '
+                     r'\[(?P<origin>.+)\]')
+class RAMLLintBear:
+    """
+    RAML Linter is a static analysis, linter-like, utility that will enforce
+    rules on a given RAML document, ensuring consistency and quality.
+    """
```

Comment:
like discussed on gitter, please put a note that files should not have leading empty newlines!